app/store/migrations/079.ts

```diff
@@ -0,0 +1,64 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { ensureValidState } from './util';
+import { captureException } from '@sentry/react-native';
+
+/**
+ * Migration 74: Reset PhishingController phishingLists
+ *
+ * This migration resets only the phishingLists array in the PhishingController state
+ * while preserving all other state properties. This allows the app to rebuild the lists
+ * while maintaining user preferences and configuration.
+ */
+const migration = (state: unknown): unknown => {
+  const migrationVersion = 74;
```

Comment:
```suggestion
  const migrationVersion = 79;
```

You have a lot of references to `74` throughout this PR - plz `ctrl + F` and fix 😉